media/css/mozorg/home/home.scss

```diff
@@ -472,7 +472,7 @@ main {
         padding-top: 40px;
 
         &:before {
-            @include at2x('/media/img/firefox/common/firefox-logo.png');
+            @include at2x('/media/img/firefox/template/logo-high-res.png');
```

Comment:
CircleCI is failing with a CSS pre-processing error: `ValueError: The file 'img/firefox/template/logo-high-res-high-res.png' could not be found`, which is happening because you're passing `logo-high-res.png` to the at2x mixin, instead of leaving it up to the mixin to do the logic. If you change this to just `logo.png` then it should work.